etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -69,13 +96,24 @@ var (
 	}
 )
 
+// ServerError defines gRPC server errors.
+// (https://github.com/grpc/grpc-go/blob/master/rpc_util.go#L319-L323)
+type ServerError struct {
+	code codes.Code
+	desc string
+}
+
+func (e ServerError) Error() string {
+	return fmt.Sprintf("rpc error: code = %d desc = %s", e.code, e.desc)
```

Comment:
Sure. So `Error` method only returns `desc` and add another method `Code` right?